goosebit/ui/templates/rollouts.html

```diff
@@ -34,41 +34,67 @@ <h5 class="modal-title">Create Rollout</h5>
                     aria-label="Close"
                 ></button>
             </div>
-            <div class="modal-body">
-                <input
-                    id="rollout-selected-name"
-                    class="form-control mb-3"
-                    placeholder="Name"
-                />
-                <input
-                    id="rollout-selected-feed"
-                    class="form-control mb-3"
-                    placeholder="Feed"
-                />
-                <input
-                    id="rollout-selected-flavor"
-                    class="form-control mb-3"
-                    placeholder="Flavor"
-                />
-                <select class="form-select" id="selected-fw"></select>
-            </div>
-            <div class="modal-footer">
-                <button
-                    type="button"
-                    class="btn btn-secondary"
-                    data-bs-dismiss="modal"
-                >
-                    Close
-                </button>
-                <button
-                    type="button"
-                    class="btn btn-outline-light"
-                    data-bs-dismiss="modal"
-                    onclick="createRollout()"
-                >
-                    Save changes
-                </button>
-            </div>
+            <form id="rollout-form" class="needs-validation" novalidate>
+                <div class="modal-body">
+                    <div class="form-group mb-3">
+                        <label for="rollout-selected-name">Name</label>
+                        <input
+                            id="rollout-selected-name"
+                            class="form-control"
+                            placeholder="Release 1"
+                        />
+                    </div>
+                    <div class="form-group mb-3">
+                        <label for="rollout-selected-feed">Feed</label>
+                        <input
+                            id="rollout-selected-feed"
+                            class="form-control"
+                            placeholder="qa"
+                            required
+                        />
+                        <div class="invalid-feedback">
+                            Feed missing. Use "default" if working with a single
+                            feed.
+                        </div>
+                    </div>
+                    <div class="form-group mb-3">
+                        <label for="rollout-selected-flavor">Flavor</label>
+                        <input
+                            id="rollout-selected-flavor"
+                            class="form-control"
+                            placeholder="debug"
```

Comment:
Placeholder seems wrong.

Reply:
"debug" would be a correct example of a flavor. But we decided to remove the "flavor" feature for an initial release -  so this will go away in a future PR.